dl_binaries.sh

```diff
@@ -15,26 +16,21 @@ fi
 
 # This script downloads the binaries for the most recent version of TabNine.
 # Infrastructure detection heavily inspired by https://github.com/tzachar/cmp-tabnine/blob/main/install.sh
-version=${version:-$(curl -sS https://update.tabnine.com/bundles/version)}
-if [ $# -gt 0 ]; then
-  # Pass fully qualified names as positional arguments
-  targets="$(printf '%s\n' "$@")"
```

Comment:
To be clear, are we removing support for passing specific architecture(s) to download to the script?

Reply:
If that is an intentional change, the powershell script should remove support as well, for symmetry (plus the current implementation requires that, since the meaning of positional arguments changes)